src/pages/Variants.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useLayoutEffect, useState } from 'react';
+import { useLocation, useParams } from 'react-router-dom';
+import { VUE } from '../model/VUE';
+import { fetchVueData } from '../utils/VUEDataFetcherUtils';
+
+export const Variants: React.FC = () => {
+    const state = useLocation().state;
+    const [vue, setVue] = useState<VUE | undefined>(state);
+
+    const gene = useParams().gene;
+
+    useEffect(() => {
+        if (!state) {
+            const fetchData = async () => {
+                const data = await fetchVueData();
+                setVue(data.find(i => i.hugoGeneSymbol === gene));
+            };
+    
+            fetchData();
+        }
+    }, [gene, state]);
+
+    useLayoutEffect(() => {
+        window.scrollTo(0, 0);
```

Comment:
why do we need this one?

Reply:
there was a problem where when going to other pages, the scroll state would remain rather than start from the top of the page. this fixes that